pkg/kwokctl/runtime/cluster_snapshot.go

```diff
@@ -38,7 +38,7 @@ func (c *Cluster) SnapshotSaveWithYAML(ctx context.Context, path string, filters
 	kubeconfigPath := c.GetWorkdirPath(InHostKubeconfigName)
 	// In most cases, the user should have full privileges on the clusters created by kwokctl,
 	// so no need to expose impersonation args to "snapshot save" command.
-	return snapshot.Save(ctx, kubeconfigPath, file, filters, rest.ImpersonationConfig{})
+	return snapshot.Save(ctx, kubeconfigPath, file, filters, rest.ImpersonationConfig{}, pageSize, pageBufferSize)
```

Comment:
Ohh, I seem to have misremembered 😄, it should be like `--as` and `--as-group`.

`--page-size` and `--page-buffer-size` are fixed defaults for `save`, while `export` can be changed